src/Allegro.DotnetSdk/Sdk/editor.globalconfig

```diff
@@ -436,12 +436,3 @@ dotnet_diagnostic.MA0056.severity = suggestion
 
 # MA0074: Avoid implicit culture-sensitive methods
 dotnet_diagnostic.MA0074.severity = suggestion
-
```

Comment:
moved somewhere or in net 8 something is different?

Reply:
I believe it was never used, as these IDs are for diagnostics from CezaryPiatek's package which is not referenced by our Sdk.